test/integration/scheduler/queueing/former/queue_test.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package queueing
+
+import (
+	"fmt"
+	"testing"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	featuregatetesting "k8s.io/component-base/featuregate/testing"
+	"k8s.io/kubernetes/pkg/features"
+	"k8s.io/kubernetes/test/integration/scheduler/queueing"
+)
+
+// TestCoreResourceEnqueue verify Pods failed by in-tree default plugins can be
+// moved properly upon their registered events.
+// Here, we run only the test cases where the EnableSchedulingQueueHint is disabled.
+func TestCoreResourceEnqueue(t *testing.T) {
+	for _, tt := range queueing.CoreResourceEnqueueTestCases {
+		if tt.EnableSchedulingQueueHint == nil || !tt.EnableSchedulingQueueHint.Has(false) {
```

Comment:
Shouldn't it be just `if !tt.EnableSchedulingQueueHint.Has(false) {`? According to the comment, `EnableSchedulingQueueHint` is {true, false} by default:

https://github.com/kubernetes/kubernetes/blob/21b8cdefc5949a8e88ba0fa6922590c9fd8c2f83/test/integration/scheduler/queueing/queue.go#L76-L78

Reply:
Ahhh, yes. Great catch..